api4/user_test.go

```diff
@@ -20,23 +20,14 @@ import (
 func TestCreateUser(t *testing.T) {
 	th := Setup().InitBasic()
 	defer th.TearDown()
-	Client := th.Client
-	AdminClient := th.SystemAdminClient
-
-	enableOpenServer := th.App.Config().TeamSettings.EnableOpenServer
-	enableUserCreation := th.App.Config().TeamSettings.EnableUserCreation
-	defer func() {
-		th.App.UpdateConfig(func(cfg *model.Config) { cfg.TeamSettings.EnableOpenServer = enableOpenServer })
-		th.App.UpdateConfig(func(cfg *model.Config) { cfg.TeamSettings.EnableUserCreation = enableUserCreation })
-	}()
 
 	user := model.User{Email: th.GenerateTestEmail(), Nickname: "Corey Hulen", Password: "hello1", Username: GenerateTestUsername(), Roles: model.SYSTEM_ADMIN_ROLE_ID + " " + model.SYSTEM_USER_ROLE_ID}
 
-	ruser, resp := Client.CreateUser(&user)
+	ruser, resp := th.Client.CreateUser(&user)
```

Comment:
Most of this consists in just eliminating the `Client := th.Client` pattern and using the variables directly for more clarity.